src/nestor/models/contacts_book.py

```diff
@@ -83,6 +100,24 @@ def find_phone(self, phone: str) -> Phone | None:
             if p.value == phone:
                 return p
         return None
+
+    def add_email(self, new_email: str) -> None:
```

Comment:
Let's not raise error here.
For Add contact we have the logic that we update contact if exist, and add if not.